tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/object/AnonymousObjectTest.java

```diff
@@ -148,12 +148,13 @@ public void testObjectWithSelfReference() {
         Assert.assertEquals(returns[1].stringValue(), "Tyler Jewell");
     }
 
-    @Test(description = "Negative test to test un-defaultable anon object")
+    @Test(description = "Negative test to test un-defaultable anon object", enabled = false)
```

Comment:
better to add to the  "brokenOnLangLibChange", rather than disabling

Reply:
Done